sentry-uninstall.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"
+
+NAMESPACE=common
+ENVIRONMENT="common/dev"  # Default environment
+
+function usage() {
+  echo "Usage: $(basename "$0") [-h] [-f] [ENVIRONMENT]"
+  echo "Uninstall sentry from $NAMESPACE"
+  echo ""
+  echo "  -h  Show this help"
+  echo "  -f  Full uninstall. Clean PersistentVolume, etc."
+  echo "  ENVIRONMENT  Target environment (default: common/dev)"
+  echo ""
+  echo "Examples:"
+  echo "  $(basename "$0")                    # Uses common/dev (default)"
+  echo "  $(basename "$0") common/qa          # Uses common/qa"
+  echo "  $(basename "$0") -f common/prod     # Full uninstall in common/prod"
```

Comment:
"common/qa" and "common/prod" are not real environments, not sure if thats intentional

Reply:
I will set real ones